sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithValueStateTTLSuite.scala

```diff
@@ -265,50 +265,129 @@ class TransformWithValueStateTTLSuite extends TransformWithStateTTLTest {
         val fm = CheckpointFileManager.create(stateSchemaPath, hadoopConf)
 
         val keySchema = new StructType().add("value", StringType)
+        val expirationSchema = new StructType().add("expirationMs", LongType)
+        val compositeKeySchema = new StructType()
+          .add("key", new StructType().add("value", StringType))
+          .add("userKey", new StructType()
+            .add("id", IntegerType, false)
+            .add("name", StringType))
+
+        // Define all 9 expected schemas
```

Comment:
Why not just define them as intuitive order (e.g. state schema and TTL schema closer) and assign CF ID later after sorting? It's probably easy to miss that the state names have to be sorted when we issue IDs for them.